mesonbuild/minstall.py

```diff
@@ -181,6 +183,9 @@ def restore_selinux_contexts():
 
 
 def get_destdir_path(d, path):
+    '''
+    here we dist directory path
```

Comment:
What does this comment mean?

Reply:
This comment was a mistake and is removed.